src/views/inline_footnote.ts

```diff
@@ -113,6 +184,10 @@ export class InlineFootnoteView<
       'footnote',
       ...getChangeClasses(this.node.attrs.dataTracked),
     ].join(' ')
+
+    if (this.isSelected()) {
+      this.activateGenericFnModal()
```

Comment:
I'm not clear on when the generic footnotes modal is expected to activate. Based on this change, wouldn't it activate even when selecting an inline footnote in a table?

Reply:
Updated to avoid opening it for empty table footnotes